config/gregtech/worldgen/beneath/precious_vein.json

```diff
@@ -1,53 +1,53 @@
-{
-  "weight": 40,
-  "density": 0.3,
-  "max_height": 120,
-  "min_height": 50,
-  "dimension_filter": [
-    "is_nether"
-  ],
-  "surface_stone_material": "gold",
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [
-      14,
-      17
-    ]
-  },
-  "filler": {
-    "type": "ga_simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 100,
-          "value": "ore:silver"
-        },
-        {
-          "weight": 20,
-          "value": "ore:poor:precious_metal"
-        },
-        {
-          "weight": 50,
-          "value": "ore:precious_metal"
-        },
-        {
-          "weight": 10,
-          "value": "ore:rich:precious_metal"
-        },
-        {
-          "weight": 5,
-          "value": "ore:pure:precious_metal"
-        },
-        {
-          "weight": 10,
-          "value": "ore:poor:gold"
-        },
-        {
-          "weight": 5,
-          "value": "ore:gold"
-        }
-      ]
-    }
-  }
+{
+  "weight": 40,
+  "density": 0.3,
+  "max_height": 120,
+  "min_height": 50,
+  "dimension_filter": [
+    "dimension_id:10"
+  ],
+  "surface_stone_material": "gold",
+  "generator": {
+    "type": "ellipsoid",
+    "radius": [
+      14,
+      17
+    ]
+  },
+  "filler": {
+    "type": "ga_simple",
+    "value": {
+      "type": "weight_random",
+      "values": [
+        {
+          "weight": 100,
+          "value": "ore:silver"
+        },
+        {
+          "weight": 20,
+          "value": "ore:poor:precious_metal"
+        },
+        {
+          "weight": 50,
+          "value": "ore:precious_metal"
+        },
+        {
+          "weight": 10,
+          "value": "ore:rich:precious_metal"
+        },
+        {
+          "weight": 5,
+          "value": "ore:pure:precious_metal"
+        },
+        {
+          "weight": 10,
+          "value": "ore:gold"
+        },
+        {
+          "weight": 5,
+          "value": "ore:rich:gold"

```

Comment:
What I really meant here was that the yield of ores from this entire vein in general used to be 2X of what it is now, since it was in the nether previously. Perhaps some kind of buff can be applied in that way, instead of giving more regular gold. The gold process byproducts are also not increased by increasing the regular gold content as well.

Reply:
i could put up each tier by one, like so poor -> normal -> rich -> pure?